### DISCUSSION TITLE
URLs hoarder does not properly work with when extracting content for read-later

### DISCUSSION THREAD

**Discussion by ovizii** [karakeep-app/karakeep#85] [Q&A]
Would you like us to submit those to a specific place, so you could check and perfect your extraction code?

**Comment by ovizii** [comment]
Small collection:
- https://github.com/DavideWiest/Reader
- https://audiomonkey.ro/products/boxe-de-raft-elac-debut-reference-dbr62?variant=47890128273743

**Comment by MohamedBassem** [comment]
What kind of problems are you facing with github?

Looks fine for me:

<img width="346" alt="Screenshot 2024-04-16 at 12 20 58 AM" src="https://github.com/MohamedBassem/hoarder-app/assets/2418637/99895b17-792a-4fe0-afd9-44fafda9dc03">

Any change you might be hitting https://github.com/MohamedBassem/hoarder-app/issues/93 ?

**Comment by ovizii** [comment]
Sorry, I should have added screenshots for the two links.

![image](https://github.com/MohamedBassem/hoarder-app/assets/837902/8f208894-d1c9-40ed-85c1-d05f89f30cf2)

![image](https://github.com/MohamedBassem/hoarder-app/assets/837902/0f9ebe23-439a-40c0-8bc4-5876fccc514f)
